### PR TITLE
nautilus: doc: reset PendingReleaseNotes following 14.2.8 release

### PR DESCRIPTION
**Pull request by smithfarm**
This commit cannot be cherry-picked from master because
PendingReleaseNotes is maintained separately for each stable release.

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -1,45 +1,3 @@
...
-* The MGR now accepts 'profile rbd' and 'profile rbd-read-only' user caps.
-  These caps can be used to provide users access to MGR-based RBD functionality
-  such as 'rbd perf image iostat' an 'rbd perf image iotop'.
-
-* The configuration value ``osd_calc_pg_upmaps_max_stddev`` used for upmap
-  balancing has been removed. Instead use the mgr balancer config
-  ``upmap_max_deviation`` which now is an integer number of PGs of deviation
-  from the target PGs per OSD.  This can be set with a command like
-  ``ceph config set mgr mgr/balancer/upmap_max_deviation 2``.  The default
-  ``upmap_max_deviation`` is 1.  There are situations where crush rules
```
Comment (dzafman) [inline PendingReleaseNotes]: Let's update this from 1 -> 5.

**Comment by neha-ojha** [comment]
@smithfarm AFAIK, we fixed `upmap_max_deviation` in https://github.com/ceph/ceph/pull/33670#discussion_r387101144, what else is left to be addressed?

**Comment by smithfarm** [comment]
@dzafman @neha-ojha The purpose of this PR is to zero out the file `PendingReleaseNotes` in nautilus, since the current contents relate to 14.2.8 which is already released.

The release notes for 14.2.8 are here:

* https://github.com/ceph/ceph/blob/master/doc/releases/nautilus.rst#v1428-nautilus
* https://ceph.io/releases/v14-2-8-nautilus-released/

Both of these say "The default `upmap_max_deviation` is 5."

What I'm trying to do here is get rid of all the old entries in PendingReleaseNotes so we can start 14.2.9 preparation with a clean slate. That way nobody will get confused about which release notes relate to 14.2.9 and which relate to older releases. The confusion arises because, when asked to add a release note to PendingReleaseNotes, folks often do not check whether they are adding it under the right heading - some append it at the beginning, others at the end of the file. At release announcement time, we have to clean it up and it can be quite tedious.

Starting 14.2.9 from a clean slate will make work with this file much easier, IMO.

**Comment by smithfarm** [comment]
@dzafman @neha-ojha @theanalyst You can see an example of the problem I referred to above in the current nautilus `PendingReleaseNotes`:

https://github.com/ceph/ceph/blob/nautilus/PendingReleaseNotes

Commit 3996479f7dcf added a note about "Kafka endpoints". This is a new feature that will be in 14.2.9 and it was added as part of the 14.2.9 release preparation stage that is currently ongoing. Yet, if you look at the document as a whole, this new release note is under the heading:

```
14.2.8
------
```

so it's very difficult to tell it apart from the preceding release notes, which relate to 14.2.8.

**Comment by theanalyst** [comment]
Agree we need to zero out the PendingReleaseNotes, @dzafman @neha-ojha, the actual content is already in 14.2.8 release notes

**Comment by smithfarm** [comment]
make check failure is unrelated:

```
The following tests FAILED:
	  5 - run-tox-mgr-dashboard (Failed)
```

**Pull request merged**